### DISCUSSION TITLE
Route "alias" or default route

### DISCUSSION THREAD

**Discussion by SeraphicRav** [remix-run/remix#5974] [Q&A] [answered]
Hello !

I am using remix in version 1.15.0.

I am trying to make the path / show the same thing as /offers/all/new/1 but I could not manage to do it. I do not want to make a redirect as it makes Google tools not happy about that. How should I proceed ?

I have a file structure like this:
![image](https://user-images.githubusercontent.com/3774520/229277397-cc414f38-8c50-4031-a671-e1a74e9c052d.png)

app/routes/__with-menus/__main/index.tsx contains:
```typescript
export * from './offers/$category/$order/$page';
export { default } from './offers/$category/$order/$page';
```
With this file, the server rendering works but it fails after some time when trying to reload data from client side.

**Comment by SeraphicRav** [comment, marked as answer]
Oh, I removed app/routes/__with-menus/__main/index.tsx and use optional paths instead for the default behavior and now it works:
![image](https://user-images.githubusercontent.com/3774520/229279548-e58587cc-e3c0-4481-afb3-92fdcf9e2abe.png)

**Reply by sergiodxa** [reply]
Remember to set a canonical url meta tag, otherwise it may be considered duplicated content to Google.

**Reply by SeraphicRav** [reply]
Thanks, it was set before I tried to solve this problem ;)
